diff_drive_controller/include/diff_drive_controller/speed_limiter.hpp

```diff
@@ -19,7 +19,7 @@
 #ifndef DIFF_DRIVE_CONTROLLER__SPEED_LIMITER_HPP_
 #define DIFF_DRIVE_CONTROLLER__SPEED_LIMITER_HPP_
 
-#include <cmath>
```

Comment:
Thanks for the hint. I used now `std::numeric_limits<double>::quiet_NaN()` instead of the c-style NANs. (and cleanedup the cmath includes in other files)